mgmt/RecordsConfig.cc

```diff
@@ -263,6 +263,8 @@ static const RecordElement RecordsConfig[] =
   ,
   {RECT_CONFIG, "proxy.config.admin.api.restricted", RECD_INT, "0", RECU_RESTART_TM, RR_NULL, RECC_INT, "[0-1]", RECA_NULL}
   ,
+  {RECT_CONFIG, "proxy.config.periodic_track_config_files", RECD_INT, "0", RECU_RESTART_TM, RR_NULL, RECC_INT, "[0-1]", RECA_NULL}
```

Comment:
I understand this is not going to be a small change, but I feel like `config_tracking_interval` that accepts 0 (no tracking) or N (second) is a more common way of configuring this kind of things on ATS.

Reply:
+1

Reply:
I will see if it is easy to do that.